packages/stores/src/queries-external/numia/pool-apr.ts

```diff
@@ -0,0 +1,62 @@
+import { KVStore } from "@keplr-wallet/common";
+import { Dec, RatePretty } from "@keplr-wallet/unit";
+import { computed, makeObservable } from "mobx";
+import { computedFn } from "mobx-utils";
+
+import { ObservableQueryExternalBase } from "../base";
+
+type PoolData = {
+  pool_id: string;
+  swap_fees: number;
+  superfluid: number;
+  osmosis: number;
+  boost: number;
+  total_apr: number;
+};
+
+/** Queries Imperator to get the market cap of the tokens. */
+export class ObservableQueryPoolAprs extends ObservableQueryExternalBase<
```

Comment:
dumb question: why aren't we using trpc for this call?

Reply:
I found this easier for returning queried values in the cells in the table. But I'm sure there's a way to mix tRPC. It will be removed when I build the table and all its data in a tRPC function

Reply:
👍